src/raftstore/store/worker/pd.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::{Arc, RwLock};
+use std::fmt::{self, Formatter, Display};
+
+use kvproto::metapb;
+
+use util::HandyRwLock;
+use util::worker::Runnable;
+use pd::PdClient;
+
+// Use an asynchronous thread to tell pd something.
+// TODO: remove enum_variant_names if we add other task later.
+#[allow(enum_variant_names)]
+pub enum Task {
+    AskChangePeer {
```

Comment:
To make it clear, We may add a type to AskChangePeer to indicate removing or adding peer.